src/components/screens/secondPassphrase/secondPassphrase.css

```diff
@@ -54,3 +62,27 @@
     margin-top: -1px;
   }
 }
+
+.passphraseConfirmation {
+  height: 445px;
+  margin-bottom: 12px;
+}
+
+.transactionResultContent {
+  display: flex;
+  margin-top: 40px;
+  align-items: center;
+  text-align: center;
+}
+
+.confirmationFooter {
+  position: absolute;
+  display: flex;
+  bottom: 20px;
+  width: 100%;
+  justify-content: center;
+}
+
+.lastStep {
+  min-width: 720px;
+}
```

Comment:
💅 This class can be replaced by https://github.com/LiskHQ/lisk-hub/blob/1a8d62a40fbdabeab04e063189d41cb536137bc9/src/components/toolbox/box/demo.js#L19

Reply:
✅